aio/deploy/helm-chart/kubernetes-dashboard/templates/ingress.yaml

```
@@ -0,0 +1,67 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{ if .Values.ingress.enabled -}}
+{{- $serviceName := include "kubernetes-dashboard.fullname" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+{{- $paths := .Values.ingress.paths -}}
+apiVersion: extensions/v1beta1
```

```suggestion
apiVersion: networking.k8s.io/v1beta1
```

This affects the [1.X chart](https://github.com/helm/charts/blob/master/stable/kubernetes-dashboard/templates/ingress.yaml#L5) as well. Would a PR to the old chart be accepted?

---

Since this API has been introduced in kube 1.14, should we support older kube versions? If not, let's do this, if so, we can add a condition to use either the old one or the new one? What do you think?

---

You could support both with a conditional like this: https://github.com/helm/charts/blob/2897f04b79ee03487d9a995ea9b8f0618953cbca/stable/prometheus/templates/server-ingress.yaml#L7-L11

I think that would be sufficient. And dont worry about my question regarding the old chart, we ended up upgrading to this one :)